storage/innobase/buf/buf0dblwr.cc

```diff
@@ -359,7 +360,8 @@ class Double_write {
     void *frame{};
     uint32_t len{};
 
-    prepare(bpage, &frame, &len);
+    file::Block *enc_block = nullptr;
```

Comment:
```suggestion
    file::block_ptr enc_block;
```
see below about `block_ptr`